src/GraphQL/Mutations/BaseAuthResolver.php

```diff
@@ -47,7 +47,7 @@ public function makeRequest(array $credentials)
                 throw new AuthenticationException(__('Authentication exception'), __('Incorrect username or password'));
             }
 
-            throw new AuthenticationException(__($decodedResponse['error']), __($decodedResponse['message']));
+            throw new AuthenticationException(__($decodedResponse['error'] ?? ''), __($decodedResponse['message']));
```

Comment:
Would be better to return some identifier key instead of an empty string. Maybe a GeneralException or something like that,